dashboard/app/controllers/experiments_controller.rb

```diff
@@ -19,4 +21,38 @@ def set_course_experiment
     )
     redirect_to '/', flash: {notice: "You have successfully joined the experiment '#{params[:experiment_name]}'."}
   end
+
+  # GET /experiments/set_single_user_experiment/:experiment_name
+  def set_single_user_experiment
+    valid_experiments = ['2018-teacher-experience']
```

Comment:
is there a better place for this constant?

Reply:
it would be more standard to define it on the class:
```ruby
  VALID_EXPERIMENTS = [...]

  # ...
  def set_single_user_experiment
```